native/membrane_videocompositor_wgpu/src/compositor/videos.rs

```diff
@@ -0,0 +1,67 @@
+use wgpu::util::DeviceExt;
+
+use super::textures::{YUVPlane, YUVTextures};
+use super::Vertex;
+
+#[rustfmt::skip]
+const INDICES: [u16; 6] = [
+    0, 1, 3, 
+    1, 2, 3
+];
+
+pub struct InputVideo {
+    textures: YUVTextures,
+    vertices: wgpu::Buffer,
+    indices: wgpu::Buffer,
+    indices_len: u32,
+}
+
+impl InputVideo {
+    pub fn new(
+        device: &wgpu::Device,
+        width: u32,
+        height: u32,
+        position: &[Vertex; 4],
```

Comment:
Maybe we should keep position in position struct, with video position in space clip coordinates ([-1, 1] range) and scene pixel coordinates (the pixel on which is located upper left corner of the video on the scene)?

Reply:
this should be resoved in #37